apps/cli/src/lib/version.ts

```
@@ -1,33 +1,28 @@
-      new URL("../../package.json", import.meta.url),
-      // Relative to executable directory (compiled)
-      join(dirname(fileURLToPath(import.meta.url)), "../../package.json"),
-      // From current working directory
-      join(process.cwd(), "package.json"),
-      // From executable's directory
-      join(dirname(process.execPath), "../package.json"),
-    ];
+// Try to import the generated version first
+let generatedVersion: string | undefined;
+try {
+  // biome-ignore lint/suspicious/noTsIgnore: Not always generated
```

Biome (2.1.2)

[error] Failed to parse category "lint/suspicious/noTsIgnore". Example of suppression: // biome-ignore lint: reason

Biome can't recognize the category, usually it's because it doesn't match the expected ones.
Example of suppression: // biome-ignore lint: reason

(suppressions/parse)